pkgs/applications/editors/vim/plugins/luaPackagePlugins.nix

```diff
@@ -0,0 +1,48 @@
+{
+  lib,
+  buildNeovimPlugin,
+  neovim-unwrapped,
+}:
+let
+  luaPackages = neovim-unwrapped.lua.pkgs;
+in
+self: super:
+(
+  let
+    luarocksPackageNames = [
+      "fidget-nvim"
+      "gitsigns-nvim"
+      "image-nvim"
+      "lsp-progress-nvim"
+      "lualine-nvim"
+      "luasnip"
+      "lush-nvim"
+      "lz-n"
+      "lze"
+      "lzextras"
+      "lzn-auto-require"
+      "middleclass"
+      "mini-test"
+      "neorg"
+      "neotest"
+      "nui-nvim"
+      "nvim-cmp"
+      "nvim-nio"
+      "nvim-web-devicons"
+      "oil-nvim"
+      "orgmode"
+      "papis-nvim"
+      "rest-nvim"
+      "rocks-config-nvim"
+      "rtp-nvim"
+      "telescope-manix"
+      "telescope-nvim"
+    ];
+    toVimPackage =
+      name:
+      buildNeovimPlugin {
+        luaAttr = luaPackages.${name};
+      };
```

Comment:
At this point, we could probably just inline this into the `genAttrs` expression? I don't think having a `toVimPackage` binding adds any readability anymore.

```nix
lib.genAttrs luarocksPackageNames (
  name:
  buildNeovimPlugin {
    luaAttr = luaPackages.${name};
  }
)
```